src/Runtime.Base/src/System/Runtime/RuntimeExports.cs

```diff
@@ -364,6 +364,13 @@ public static unsafe ushort RhGetComponentSize(EETypePtr ptrEEType)
             return pEEType->ComponentSize;
         }
 
+        [RuntimeExport("RhGetBaseSize")]
+        public static unsafe uint RhGetBaseSize(EETypePtr ptrEEType)
```

Comment:
I'm all up for that. Moving the managed definition of src/Runtime.Base/src/System/Runtime/EEType.cs out to src/Common/... also unblocks unit testability of EEType emission (now we're free to include that file from a unit test and use it to interrogate a byte buffer prepared by the EEType emitter).

Filed #1268 to track that work.